src/main/java/seedu/address/logic/commands/patient/ClearCommand.java

```diff
@@ -13,11 +16,10 @@
     public static final String COMMAND_WORD = "clear";
```

Comment:
Since AddCommand was changed to AddPatientCommand, maybe ClearCommand should be changed to be more descriptive also? i.e. change to ClearAllPatientsCommand.